interpreter/runtime/table.ml

```diff
@@ -17,47 +17,62 @@ exception OutOfMemory
 let valid_limits {min; max} =
   match max with
   | None -> true
-  | Some m -> I32.le_u min m
+  | Some m -> I64.le_u min m
 
-let create size r =
-  try Lib.Array32.make size r
+let valid_index it i =
+  match it with
+  | I32IndexType -> I64.le_u i 0xffff_ffffL
+  | I64IndexType -> true
+
+let create size it r =
```

Comment:
Why do we need it here?

Reply:
Oh I guess maybe we don't.  I think I just copied that from the memory.ml.  Fixed.